### PR TITLE
Add constraint on inmanta-module-yum

### PR DESCRIPTION
**Pull request by hlloreda**
# Description

Added support for inmanta-module-yum 2.0.0

note: to add a changelog entry and bump the version number:
`inmanta module release --dev [--major|--minor|--patch] [--changelog-message "<your_changelog_message>"]`

# [Merge procedure](https://internal.inmanta.com/development/core/tasks/commiting-changes-modules.html)

1. merge using the merge button
2. tag and bump

```sh
git checkout master
git pull
inmanta module release
git push
git push {tag} # push the tag as well
```
3. Remove the branch

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [ ] Attached issue to pull request
- [x] Changelog entry
- [ ] Version number is bumped to dev version
- [ ] Code is clear and sufficiently documented
- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Correct, in line with design
- [ ] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )

### CODE REVIEW

File: model/_init.cf [hunk 1]
```diff
@@ -94,7 +94,7 @@ implementation postgresqlServer for PostgresqlServer:
         content=std::template('postgresql/postgresql.conf.j2'),
         requires=[postgresql_initdb_script],
     )
-    postgresql_service = systemd::Service(name="postgresql-{{ pg_version }}", host=self.host, state="running", onboot=true,
```
Comment (arnaudsjs) [inline model/_init.cf:97]: I think you incorrectly removed the state attribute from the `systemd:Service` entity while we should only do that for the `yum:Package` entities.

**Review by arnaudsjs — CHANGES_REQUESTED** [review]
This module is not compatible with `inmanta-module-yum<2`, so we need to add a version constraint on the `inmanta-module-yum` package to make sure pip doesn't install an incompatible version of the yum module.

**Review by arnaudsjs — CHANGES_REQUESTED** [review]
I don't understand this PR. The model uses `yum::package`, which is only available in `inmanta-module-yum>=2`, but this PR adds the constraint `inmanta-module-yum<2`. So the resulting module is not going to work. I think your original PR was ok, it was just missing a constraint to make sure it would not run against `inmanta-module-yum<2`.

**Comment by hlloreda** [comment]
In the near future, this module will be archived. Therefore, this pull request is not necessary.

**Pull request closed without merging**